---
layout: about
title: about
permalink: /
description: Postdoctoral Associate, <a href="https://chunglab.org" target="_blank">Chung Lab</a> at MIT.

profile:
  align: right
  image: profile.jpg
  address: >
    <p>E-mail:</p>
    <p>[firstname][lastname] AT mit.edu</p>

news: true  # includes a list of news items
selected_papers: false # includes a list of papers marked as "selected={true}"
social: true  # includes social icons at the bottom of the page
---

I'm a computational biologist who specializes in sequencing and imaging data analysis. I majored in Biology for my undergraduate at Seoul National University, and then joined [Narry Kim Lab](https://narrykim.org){:target="_blank"} for my Ph.D. to study post-transcriptional regulations of coding and non-coding RNAs. During my graduate period, I'd got trained to do both wet and computational work with an emphasis on the latter. Besides typical sequencing methods such as (sc)RNA-seq, I have extensive experience in library preparation and data analysis for some specialized custom RNA-seq datasets, for example, TAIL-seq for measuring the global mRNA poly(A) tail length.

After I joined Chung Lab as a postdoc, I've been focusing on microscopy imaging data analysis. The typical datasets that I analyze include large-scale confocal/SPIM images of enlarged and cleared tissue-gel hybrids, which usually demand dedicated machine learning techniques and efficient parallelization of computation.

My current research interest lies in spatial transcriptomics/proteomics in the neuroscience and immunology contexts. I believe we could obtain unprecedentedly rich information at multiple scales if we could combine and utilize the strengths of both sequencing and imaging techniques.

---
