---
layout: about
title: about
permalink: /
description: Postdoctoral Associate, the <a href="https://chunglab.org" target="_blank">Chung lab</a> at MIT.

profile:
  align: right
  image: profile.jpeg
  address: >
    <p>E-mail:</p>
    <p>[firstname][lastname] AT mit.edu</p>

news: true  # includes a list of news items
selected_papers: false # includes a list of papers marked as "selected={true}"
social: true  # includes social icons at the bottom of the page
---

I'm a computational biologist who specializes in sequencing and imaging data analysis. I majored in Biology for my undergraduate at Seoul National University and then joined the [Narry Kim lab](https://narrykim.org){:target="_blank"} to study RNA biology.
During my Ph.D., I carried out computational analysis on varous types of next-generation sequencing (NGS) datasets.
Besides standard NGS technologies such as bulk or single-cell RNA-seq, I have extensive experience in preparing and analyzing custom RNA-seq libraries. For example, I used [TAIL-seq](https://www.illumina.com/science/sequencing-method-explorer/kits-and-arrays/tail-seq.html){:target="_blank"} to profile global poly(A)-tail length to study mRNA poly(A) tail regulations, and [fCLIP-seq](https://www.sciencedirect.com/science/article/abs/pii/S1046202318300641?via%3Dihub){:target="_blank"} to reveal the endogenously-expressed dsRNA binding repertoire of PKR.

After I joined the Chung lab as a postdoc, I've been focusing on hydrogel-based tissue engineering ([review](https://www.nature.com/articles/s41583-019-0250-1){:target="_blank"}) and microscopy imaging data analysis. I developed [eMAP](https://science.org/doi/10.1126/sciadv.abf6589){:target="_blank"}, a synthetic gel-based tissue-transforming method that maximally preserves antigenicity by purely physical tissue-hydrogel hybridization. It enables superresolution imaging with a large library of commercial antibodies and conventional confocal microscopy. Typical datasets I analyze are large-scale confocal/light-sheet microscopy images of cleared and expanded tissue-gel hybrids, which usually require dedicated machine learning techniques and efficient parallelization of computation for image processing. My current aim is to generate a single-cell mouse brain atlas and develop an end-to-end 3D volumetric image analysis framework to facilitate the spatial interrogation of biological systems.

My long-term research interest lies in single-cell and spatial genomics in the neuroscience and immunology contexts. I'm particularly interested in how cell-to-cell heterogeneity develops within the healthy and diseased tissue microenvironment. I believe cross-disciplinary approaches utilizing both sequencing and imaging techniques would open an avenue for investigating biological tissues in greater detail at multiple scales.

---
