<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>Joha  Park | publications</title>
<meta name="description" content="A Joha Park's personal webpage.
">

<!-- Open Graph -->


<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">
<!-- <link rel="preconnect" href="https://fonts.gstatic.com"> -->
<link href="https://fonts.googleapis.com/css2?family=Syne+Mono&display=swap" rel="stylesheet">

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

<!-- Styles -->
<link rel="shortcut icon" href="/assets/img/favicon.ico">
<link rel="stylesheet" href="/assets/css/main.css">

<link rel="canonical" href="/publications/">

<!-- JQuery -->
<!-- jQuery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>


<!-- Theming-->

  <script src="/assets/js/theme.js"></script>
  <!-- Load DarkMode JS -->
<script src="/assets/js/dark_mode.js"></script>



  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-KVZHRXHHWE"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'G-KVZHRXHHWE');
  </script>




    
<!-- MathJax -->
<script type="text/javascript">
  window.MathJax = {
    tex: {
      tags: 'ams'
    }
  };
</script>
<script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.1.2/es5/tex-mml-chtml.js"></script>
<script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
    <div class="container">
      
      <a class="navbar-brand title font-weight-lighter" href="/">
       <span class="font-weight-bold">Joha</span>   Park
      </a>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">
              about
              
            </a>
          </li>
          

          <!-- CV -->
          <li class="nav-item ">
            <a class="nav-link" href="/assets/pdf/cv.pdf">
              cv
            </a>
          </li>

          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          
          
          
          
          <li class="nav-item active">
              <a class="nav-link" href="/publications/">
                publications
                
                <span class="sr-only">(current)</span>
                
              </a>
          </li>
          
          
          
            <div class = "toggle-container">
              <a id = "light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
              </a>
            </div>
          
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    <h1 class="post-title">publications</h1>
    <p class="post-description"></p>
  </header>

  <article>
    <p>*: co-first, #: co-correspondence</p>
<div class="publications">


  <h2 class="year">2023</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="Park2023" class="col-sm-8">
    
      <div class="title">Short poly(A) tails are protected from deadenylation by the LARP1-PABP complex</div>
      <div class="author">
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                <em>J. Park*</em>,
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  M. Kim*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  H. Yi*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  K. Baeg*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  Y. Choi,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  Y.-s. Lee,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  J. Lim,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  and V. N. Kim
                
              
            
          
        
      </div>

      <div class="periodical">
      
        
          <em>Nat. Struct. Mol. Biol.</em>, 30(3):330–338
        
      
      <!-- 
        , 2023
       -->
      </div>
      <div>
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1038/s41594-023-00930-y" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Deadenylation generally constitutes the first and pivotal step in eukaryotic mRNA decay. Despite its importance in post-transcriptional regulations, the kinetics of deadenylation and its regulation remain largely unexplored. Here, we identify LARP1 as a general decelerator of deadenylation, which acts mainly in the 30–60-nt poly(A) length window. We measured the steady-state and pulse-chased distribution of poly(A) tail length, and found that deadenylation slows down in the 30–60 nt range. LARP1 associates preferentially with short tails and its depletion results in accelerated deadenylation specifically in the 30–60 nt range. Consistently, LARP1 knockdown leads to a global reduction of mRNA abundance. LARP1 interferes with the CCR4-NOT-mediated deadenylation in vitro by forming a ternary complex with PABP and poly(A). Together, our work reveals a dynamic nature of deadenylation kinetics and a role of LARP1 as a poly(A) length-specific barricade that creates a threshold for deadenylation.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2021</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="Park2021" class="col-sm-8">
    
      <div class="title">Epitope-preserving Magnified Analysis of Proteome (eMAP)</div>
      <div class="author">
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                <em>J. Park*</em>,
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  S. Khan*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  D. H. Yun*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  T. Ku,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  K. L. Villa,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  J. E. Lee,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  Q. Zhang,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  J. Park,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  G. Feng,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  E. Nedivi#,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  and K. Chung#
                
              
            
          
        
      </div>

      <div class="periodical">
      
        
          <em>Sci. Adv.</em>, 7(46):eabf6589
        
      
      <!-- 
        , 2021
       -->
      </div>
      <div>
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://www.science.org/doi/full/10.1126/sciadv.abf6589?adobe_mc=MCMID%3D44649648719534898001840095578056377326%7CMCORGID%3D242B6472541199F70A4C98A6%2540AdobeOrg%7CTS%3D1636986534" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Synthetic tissue-hydrogel methods have enabled superresolution investigation of biological systems using diffraction-limited microscopy. However, chemical modification by fixatives can cause loss of antigenicity, limiting molecular interrogation of the tissue gel. Here, we present epitope-preserving magnified analysis of proteome (eMAP) that uses purely physical tissue-gel hybridization to minimize the loss of antigenicity while allowing permanent anchoring of biomolecules. We achieved success rates of 96% and 94% with synaptic antibodies for mouse and marmoset brains, respectively. Maximal preservation of antigenicity allows imaging of nanoscopic architectures in 1000-fold expanded tissues without additional signal amplification. eMAP-processed tissue gel can endure repeated staining and destaining without epitope loss or structural damage, enabling highly multiplexed proteomic analysis. We demonstrated the utility of eMAP as a nanoscopic proteomic interrogation tool by investigating molecular heterogeneity in inhibitory synapses in the mouse brain neocortex and characterizing the spatial distributions of synaptic proteins within synapses in mouse and marmoset brains.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2018</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="Kim2018" class="col-sm-8">
    
      <div class="title">PKR Senses Nuclear and Mitochondrial Signals by Interacting with Endogenous Double-Stranded RNAs.</div>
      <div class="author">
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  Y. Kim*#,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                <em>J. Park*</em>,
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  S. Kim*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  M. Kim,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  M.-G. Kang,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  C. Kwak,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  M. Kang,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  B. Kim,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  H.-W. Rhee,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  and V. N. Kim#
                
              
            
          
        
      </div>

      <div class="periodical">
      
        
          <em>Mol. Cell</em>, 71(6):1051–1063.e6
        
      
      <!-- 
        , 2018
       -->
      </div>
      <div>
      
        - Selected as "Research Highlights" in <em>Nat. Chem. Biol.</em>, 14(11):989 (<a href="https://www.nature.com/articles/s41589-018-0159-4" target="_blank">link</a>)
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1016/j.molcel.2018.07.029" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Protein kinase RNA-activated (PKR) induces immune response by sensing viral double-stranded RNAs (dsRNAs). However, growing evidence suggests that PKR can also be activated by endogenously expressed dsRNAs. Here, we capture these dsRNAs by formaldehyde-mediated crosslinking and immunoprecipitation sequencing and find that various noncoding RNAs interact with PKR. Surprisingly, the majority of the PKR-interacting RNA repertoire is occupied by mitochondrial RNAs (mtRNAs). MtRNAs can form intermolecular dsRNAs owing to bidirectional transcription of the mitochondrial genome and regulate PKR and eIF2α phosphorylation to control cell signaling and translation. Moreover, PKR activation by mtRNAs is counteracted by PKR phosphatases, disruption of which causes apoptosis from PKR overactivation even in uninfected cells. Our work unveils dynamic regulation of PKR even without infection and establishes PKR as a sensor for nuclear and mitochondrial signaling cues in regulating cellular metabolism.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="Yi2018" class="col-sm-8">
    
      <div class="title">PABP Cooperates with the CCR4-NOT Complex to Promote mRNA Deadenylation and Block Precocious Decay.</div>
      <div class="author">
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  H. Yi*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                <em>J. Park*</em>,
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  M. Ha,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  J. Lim,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  H. Chang,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  and V. N. Kim
                
              
            
          
        
      </div>

      <div class="periodical">
      
        
          <em>Mol. Cell</em>, 70(6):1081–1088.e5
        
      
      <!-- 
        , 2018
       -->
      </div>
      <div>
      
        - Previewed in <em>Mol. Cell</em>, 70(6):987–988 (<a href="https://www.cell.com/molecular-cell/fulltext/S1097-2765(18)30451-9?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276518304519%3Fshowall%3Dtrue" target="_blank">link</a>)
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1016/j.molcel.2018.05.009" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Multiple deadenylases are known in vertebrates, the PAN2-PAN3 (PAN2/3) and CCR4-NOT (CNOT) complexes, and PARN, yet their differential functions remain ambiguous. Moreover, the role of poly(A) binding protein (PABP) is obscure, limiting our understanding of the deadenylation mechanism. Here, we show that CNOT serves as a predominant nonspecific deadenylase for cytoplasmic poly(A)+ RNAs, and PABP promotes deadenylation while preventing premature uridylation and decay. PAN2/3 selectively trims long tails (&gt;∼150 nt) with minimal effect on transcriptome, whereas PARN does not affect mRNA deadenylation. CAF1 and CCR4, catalytic subunits of CNOT, display distinct activities: CAF1 trims naked poly(A) segments and is blocked by PABPC, whereas CCR4 is activated by PABPC to shorten PABPC-protected sequences. Concerted actions of CAF1 and CCR4 delineate the ∼27 nt periodic PABPC footprints along shortening tail. Our study unveils distinct functions of deadenylases and PABPC, re-drawing the view on mRNA deadenylation and regulation.</p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="Nguyen2018" class="col-sm-8">
    
      <div class="title">Microprocessor depends on hemin to recognize the apical loop of primary microRNA.</div>
      <div class="author">
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  T. A. Nguyen*#,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                <em>J. Park*</em>,
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  T. L. Dang,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  Y.-G. Choi,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  and V. N. Kim#
                
              
            
          
        
      </div>

      <div class="periodical">
      
        
          <em>Nucleic Acids Res.</em>, 46(11):5726–5736
        
      
      <!-- 
        , 2018
       -->
      </div>
      <div>
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1093/nar/gky248" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Microprocessor, which consists of a ribonuclease III DROSHA and its cofactor DGCR8, initiates microRNA (miRNA) maturation by cleaving primary miRNA transcripts (pri-miRNAs). We recently demonstrated that the DGCR8 dimer recognizes the apical elements of pri-miRNAs, including the UGU motif, to accurately locate and orient Microprocessor on pri-miRNAs. However, the mechanism underlying the selective RNA binding remains unknown. In this study, we find that hemin, a ferric ion-containing porphyrin, enhances the specific interaction between the apical UGU motif and the DGCR8 dimer, allowing Microprocessor to achieve high efficiency and fidelity of pri-miRNA processing in vitro. Furthermore, by generating a DGCR8 mutant cell line and carrying out rescue experiments, we discover that hemin preferentially stimulates the expression of miRNAs possessing the UGU motif, thereby conferring differential regulation of miRNA maturation. Our findings reveal the molecular action mechanism of hemin in pri-miRNA processing and establish a novel function of hemin in inducing specific RNA-protein interaction.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2015</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="You2015" class="col-sm-8">
    
      <div class="title">Role of the small subunit processome in the maintenance of pluripotent stem cells.</div>
      <div class="author">
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  K. T. You,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                <em>J. Park</em>,
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  and V. N. Kim
                
              
            
          
        
      </div>

      <div class="periodical">
      
        
          <em>Genes Dev.</em>, 29(19):2004–2009
        
      
      <!-- 
        , 2015
       -->
      </div>
      <div>
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1101/gad.267112.115" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>RNA-binding proteins (RBPs) play integral roles in gene regulation, yet only a small fraction of RBPs has been studied in the context of stem cells. Here we applied an RNAi screen for RBPs in mouse embryonic stem cells (ESCs) and identified 16 RBPs involved in pluripotency maintenance. Interestingly, six identified RBPs, including Krr1 and Ddx47, are part of a complex called small subunit processome (SSUP) that mediates 18S rRNA biogenesis. The SSUP components are preferentially expressed in stem cells and enhance the global translational rate, which is critical to sustain the protein levels of labile pluripotency factors such as Nanog and Esrrb. Furthermore, the SSUP proteins are required for efficient reprogramming of induced pluripotent stem cells. Our study uncovers the role of the SSUP and the importance of translational control in stem cell fate decision. </p>
    </div>
    
  </div>
</div>
</li>
<li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="Nguyen2015" class="col-sm-8">
    
      <div class="title">Functional Anatomy of the Human Microprocessor.</div>
      <div class="author">
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  T. A. Nguyen*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  M. H. Jo*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  Y.-G. Choi,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                <em>J. Park</em>,
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  S. C. Kwon,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  S. Hohng,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  V. N. Kim#,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  and J.-S. Woo#
                
              
            
          
        
      </div>

      <div class="periodical">
      
        
          <em>Cell</em>, 161(6):1374–1387
        
      
      <!-- 
        , 2015
       -->
      </div>
      <div>
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1016/j.cell.2015.05.010" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>MicroRNA (miRNA) maturation is initiated by Microprocessor composed of RNase III DROSHA and its cofactor DGCR8, whose fidelity is critical for generation of functional miRNAs. To understand how Microprocessor recognizes pri-miRNAs, we here reconstitute human Microprocessor with purified recombinant proteins. We find that Microprocessor is an ∼364 kDa heterotrimeric complex of one DROSHA and two DGCR8 molecules. Together with a 23-amino acid peptide from DGCR8, DROSHA constitutes a minimal functional core. DROSHA serves as a "ruler" by measuring 11 bp from the basal ssRNA-dsRNA junction. DGCR8 interacts with the stem and apical elements through its dsRNA-binding domains and RNA-binding heme domain, respectively, allowing efficient and accurate processing. DROSHA and DGCR8, respectively, recognize the basal UG and apical UGU motifs, which ensure proper orientation of the complex. These findings clarify controversies over the action mechanism of DROSHA and allow us to build a general model for pri-miRNA processing.</p>
    </div>
    
  </div>
</div>
</li></ol>

  <h2 class="year">2013</h2>
  <ol class="bibliography"><li><div class="row">
  <div class="col-sm-2 abbr">
  
  </div>

  <div id="Kim2013" class="col-sm-8">
    
      <div class="title">TALEN-based knockout library for human microRNAs.</div>
      <div class="author">
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  Y.-K. Kim*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  G. Wee*,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                <em>J. Park</em>,
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  J. Kim,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  D. Baek,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  J.-S. Kim#,
                
              
            
          
        
          
          
          
          
          

          
          

          
          

          
          

          
          
          
          
          
          
          

          
            
              
                
                  and V. N. Kim#
                
              
            
          
        
      </div>

      <div class="periodical">
      
        
          <em>Nat. Struct. Mol. Biol.</em>, 20(12):1458–1464
        
      
      <!-- 
        , 2013
       -->
      </div>
      <div>
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
      <a href="https://doi.org/10.1038/nsmb.2701" class="btn btn-sm z-depth-0" role="button" target="_blank">HTML</a>
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Various technical tools have been developed to probe the functions of microRNAs (miRNAs), yet their application has been limited by low efficacy and specificity. To overcome the limitations, we used transcription activator-like effector nucleases (TALENs) to knock out human miRNA genes. We designed and produced a library of 540 pairs of TALENs for 274 miRNA loci, focusing on potentially important miRNAs. The knockout procedure takes only 2-4 weeks and can be applied to any cell type. As a case study, we generated knockout cells for two related miRNAs, miR-141 and miR-200c, which belong to the highly conserved miR-200 family. Interestingly, miR-141 and miR-200c, despite their overall similarity, suppress largely nonoverlapping groups of targets, thus suggesting that functional miRNA-target interaction requires strict seed-pairing. Our study illustrates the potency of TALEN technology and provides useful resources for miRNA research. </p>
    </div>
    
  </div>
</div>
</li></ol>


</div>

  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0">
    &copy; Copyright 2025 Joha  Park.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.

    
    
  </div>
</footer>



  </body>

  <!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>

  
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="/assets/js/mansory.js" type="text/javascript"></script>


  


<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>


</html>
